go/host/host.go

```diff
@@ -716,6 +699,47 @@ func (a *Node) monitorBlocks() {
 	subs.Unsubscribe()
 }
 
+func (a *Node) catchupMissedBlocks(lastKnownBlkHash gethcommon.Hash) {
+	var lastBlkNumber *big.Int
+	var reingestBlocks []*types.Block
+
+	// get the tip block
+	lastBlk, err := a.ethClient.BlockByNumber(lastBlkNumber)
+	if err != nil {
+		log.Panic("catching up on missed blocks, unable to fetch tip block - reason: %s", err)
+	}
+
+	if lastBlk.Hash().Hex() == lastKnownBlkHash.Hex() {
+		// no new blocks have been issued
+		return
+	}
+	reingestBlocks = append(reingestBlocks, lastBlk)
+
+	// iterate from the tip (last block produced) to the last block ingested by the node
```

Comment:
perhaps 'processed' or 'ingested' instead of 'produced'?

Reply:
```suggestion
	// iterate from the blockchain tip to the last block sucessfully ingested by the node
```

Wdyt @BedrockSquirrel ?